packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.story.tsx

```diff
@@ -19,6 +27,14 @@ const meta: ComponentMeta<typeof DateTimePicker> = {
     onChange: {
       table: { disable: true },
     },
+    minDate: { control: 'date' },
+    maxDate: { control: 'date' },
+    showSeconds: { control: 'boolean' },
```

Comment:
Do we want to add the other 3 disabledX props to storybook as well? Or at least another story where hours outside of "office hours" are disabled?

Reply:
added a new story, thank you!